plugin.rb

```diff
@@ -75,7 +75,7 @@
   # using Discourse "Topic Created" event to trigger a save.
   # `opts[]` is how you pass the data back from the frontend into Rails
   on(:topic_created) do |topic, opts, user|
-    topic.update_column(:is_resolution, true) if opts[:is_resolution]
+    topic.update_columns(is_resolution: true, closed: true) if opts[:is_resolution]
```

Comment:
there is no dependence on `communitarian resolutions close` variable?

Reply:
`communitarian_resolutions_close` is about closing and reopening resolutions automatically, isn't it?